rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/geo/TbGpsGeofencingFilterNode.java

```diff
@@ -57,14 +59,20 @@
                 "</br></br>" +
                 "{\"latitude\":  48.198618758582384, \"longitude\": 24.65322245153503, \"radius\":  100.0, \"radiusUnit\": \"METER\" }" +
                 "</br></br>" +
-                "Available radius units: METER, KILOMETER, FOOT, MILE, NAUTICAL_MILE;",
+                "Available radius units: METER, KILOMETER, FOOT, MILE, NAUTICAL_MILE;<br><br>" +
+                "Output connections: <code>True</code>, <code>False</code>, <code>Failure</code>",
         uiResources = {"static/rulenode/rulenode-core-config.js"},
         configDirective = "tbFilterNodeGpsGeofencingConfig")
 public class TbGpsGeofencingFilterNode extends AbstractGeofencingNode<TbGpsGeofencingFilterNodeConfiguration> {
 
+    @Override
```

Comment:
mistakenly add this in the commit for renaming FetchTo to TbMsgSource. Will fix this in the next commit.